crates/server/src/admin/service.rs

```diff
@@ -118,6 +120,14 @@ pub(crate) fn setup(
             "/add-client-key",
             post(handlers::add_client_key::add_client_key_handler),
         )
+        .route(
+            "/refresh-jwt-token",
+            post(handlers::add_client_key::refresh_jwt_token_handler),
+        )
+        .route(
+            "/generate-jwt-token",
+            post(handlers::add_client_key::generate_jwt_token_handler),
+        )
```

Comment:
After you finish testing move this to protected routes as this will be called from the admin dashboard and it shouldn't be called from anywhere else